quadra/tasks/classification.py

```diff
@@ -178,7 +179,7 @@ def module(self, module_config):
         )
         if self.checkpoint_path is not None:
             log.info("Loading model from lightning checkpoint: %s", self.checkpoint_path)
-            module = module.load_from_checkpoint(
```

Comment:
Why is this line needed?

Reply:
https://lightning.ai/docs/pytorch/stable/common/checkpointing_basic.html

They changed the way lightning checkpoints are imported so that it can't be called by an already instantiated model, it's only ok if calling directly from the class

model = MyLightningModule.load_from_checkpoint("/path/to/checkpoint.ckpt")